tests/tests_ntp.yml

```diff
@@ -12,6 +12,8 @@
         iburst: yes
         minpoll: 4
         maxpoll: 6
+    timesync_ntp_custom_settings:
```

Comment:
Please update README.md with this parameter

Reply:
Ok, README section in place!